deselected_tests.yaml

```diff
@@ -60,6 +60,31 @@ deselected_tests:
   - linear_model/tests/test_logistic.py::test_warm_start[multinomial-False-True-newton-cg]
   - linear_model/tests/test_logistic.py::test_warm_start[multinomial-False-True-lbfgs]
 
+  # Linear Regression - equation system is bad defined
+  - ensemble/tests/test_stacking.py::test_stacking_prefit[StackingRegressor-DummyRegressor-predict-final_estimator1-X1-y1]
+
+  # Linear Regression - no warning on passing sparse array
+  - linear_model/tests/test_base.py::test_linear_regression_pd_sparse_dataframe_warning
+
+  # Linear Regression - unsupported data type
+  - linear_model/tests/test_ransac.py::test_ransac_score
+  - linear_model/tests/test_ransac.py::test_ransac_predict
+  - linear_model/tests/test_ransac.py::test_perfect_horizontal_line
+  - model_selection/tests/test_search.py::test_grid_search_pipeline_steps
+  - tests/test_common.py::test_estimators[RANSACRegressor(estimator=LinearRegression())-check_estimators_dtypes]
+  - tests/test_multiclass.py::test_ovr_multiclass
+  - tests/test_multiclass.py::test_ovr_multilabel
+  - tests/test_multiclass.py::test_ovr_binary
```

Comment:
These tests are too important to deselect them

Reply:
Fixed